### PR TITLE
doc: wifi: Update memory footprint tables for 2.7 release

### PR DESCRIPTION
**Pull request by richabp**
Updated Wi-Fi memory footprint tables for the 2.7 release.

SHEL-2785

### CODE REVIEW

**Comment by richabp** [comment]
Please note that the values for MQTT, HTTPS server, and Location samples are under investigation.

**Comment by richabp** [comment]
> Please note that the values for MQTT, HTTPS server, and Location samples are under investigation.



> Please note that the values for MQTT, HTTPS server, and Location samples are under investigation.

Updated the values as per the latest build.

File: doc/nrf/protocols/wifi/advanced_modes/mem_requirements_raw.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      +====================================================+=============+===========================================+============================+======================+=================================+====================+======================+
+      | :ref:`Raw TX packet <wifi_raw_tx_packet_sample>`   |         507 |                                        54 |                         69 |                  180 |                             298 |                207 |                   13 |
+      +----------------------------------------------------+-------------+-------------------------------------------+----------------------------+----------------------+---------------------------------+--------------------+----------------------+
+      | :ref:`Monitor <wifi_monitor_sample>`               |         495 |                                        51 |                         69 |                  180 |                             398 |                306 |                   13 |
```
Comment (krish2718) [inline doc/nrf/protocols/wifi/advanced_modes/mem_requirements_raw.rst:23]: WPA supplicant isn't involved in monitor mode, I have raised it with the team, a PR will be raised to fix this and then we can update this row.

Comment (richabp) [reply doc/nrf/protocols/wifi/advanced_modes/mem_requirements_raw.rst:23]: I think we can update the values in the doc improvement phase after this PR is merged.

**Review by krish2718 — APPROVED** [review]

**Review by bama-nordic — APPROVED** [review]

**Review by visinor — APPROVED** [review]

**Pull request merged**